### PR TITLE
fix: Avoid allocation in the stringifyValue

### PR DESCRIPTION
**Pull request by alexandear**
The PR refactors implementation of the `stringifyValue` function.

`w.Write([]byte("<nil>"))` may allocate, better is to use `w.WriteString("<nil>")`

### CODE REVIEW

**Review by gmlewis — APPROVED** [review]
Thank you, @alexandear !
LGTM.

Awaiting second LGTM+Approval from any other contributor to this repo before merging.

**Review by air-hand — APPROVED** [review]
LGTM

**Comment by gmlewis** [comment]
Thank you, @air-hand !
Merging.

**Pull request merged**
